components/partials/ActionLink.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <a :class="[colorItem]" href="#">
+  <span class="tekstStyl">
+  {{ linktext }}
+  </span>
+  <span :class="[iconClass, colorIcon]" class="ikonaStyl"></span>
+  </a>
+</template>
+
+<script>
+export default {
+  props: {
+    linktext: {
+      type: String,
+      default: 'puste'
+    },
+    iconClass: {
+      type: String
+    },
+    colorIcon: {
+      type: String
+    },
+    colorItem: {
+      type: String
+    }
+  }
+}
+</script>
+
+<style lang="scss">
+.firstIconColor {
+  background-color: #494949;
+  color: #F3C000;
+  align-self: flex-start;
+
+  &:before {
+    font-size: 4vw;
+  }
+}
+
+.firstItemColor {
+  background-color: #494949;
+}
+
+.secondIconColor {
+  background-color: #33CB64;
+  color: #51E37A;
+  align-self: flex-end;
+
+  &:before {
+    font-size: 4vw;
+  }
+}
+
+.secondItemColor {
+  background-color: #33CB64;
+}
+
+.thirdIconColor {
+  background-color: #FD8D46;
+  color:#FADCC2;
+  align-self: flex-start;
+
+  &:before {
+    font-size: 4vw;
+  }
+}
+
+.thirdItemColor {
+  background-color: #FD8D46;
+}
+
+.forthIconColor {
+  background-color: #5A86C3;
+  color: #A9BFDE;
+  align-self: flex-end;
+
+  &:before {
+    font-size: 4vw;
```

Comment:
why vw?

Reply:
Viewpoint width are font sizes that resize with the window. If I don't use it, the icons will not resize while re-sizing the window browers, so in that case the icons will not fit in blocks.

Reply:
@zrebcu411 do we approve it?